src/buildfile.js

```diff
@@ -57,7 +57,8 @@ exports.workbenchDesktop = [
 	createModuleDescription('vs/workbench/contrib/debug/node/telemetryApp'),
 	createModuleDescription('vs/platform/files/node/watcher/watcherMain'),
 	createModuleDescription('vs/platform/terminal/node/ptyHostMain'),
-	createModuleDescription('vs/workbench/api/node/extensionHostProcess')
+	createModuleDescription('vs/workbench/api/node/extensionHostProcess'),
+	createModuleDescription('vs/workbench/contrib/issue/electron-sandbox/issueReporterMain'),
```

Comment:
To ensure we are not breaking the issue reporter, I would suggest to run a real insiders build from your branch, open it and try to open the issue reporter. When we build, bundle, minify, etc. we might hit an issue with this change that we would otherwise not see running out of sources.

Reply:
yep, I was thinking this as well. building right now on the upstream https://github.com/microsoft/vscode/tree/justin/web-issues (since i'm in a fork)

Reply:
I saw that fork and was wondering why you did not pick a branch from the main repo.